### PR TITLE
upgrade deps

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,14 +21,14 @@
   "author": "Hendrik Muhs",
   "license": "MIT",
   "dependencies": {
-    "@actions/cache": "^1.0.4",
-    "@actions/core": "^1.2.6",
-    "@actions/exec": "^1.0.4",
-    "@actions/io": "^1.0.2"
+    "@actions/cache": "^1.0.7",
```
Comment (graingert) [inline package.json:24]: https://github.com/actions/toolkit/blob/main/packages/cache/RELEASES.md#105

I'm having some trouble on the windows cpython builds

**Comment by graingert** [comment]
@hendrikmuhs can you enable dependabot btw?

**Comment by hendrikmuhs** [comment]
I am giving it a try, if I understand correctly dependabot will create a similar PR to yours?

**Comment by hendrikmuhs** [comment]
I configured dependabot, updated all dependencies, recompiled and created v1.0.5

Therefor this PR can be closed. Nevertheless, thanks for the suggestion @graingert !

**Pull request closed without merging**